bin/index.js

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Does this hash bang command need to be added to the TS file so it goes into the final transpiled code.  Or is the idea that we want to support multiple runtimes and having the `node` there is not correct?

Reply:
It's added automatically by `@deno/dnt` at the compiled output.

Reply:
Silly question, but does it add deno as the runtime or node?

Reply:
Definitly it adds `node` as the whole point of `@deno/dnt` is to compile a package for NodeJs. 

<img width="375" alt="image" src="https://github.com/user-attachments/assets/1a991130-2e78-4724-8ee2-255282dd026b" />

Reply:
Ahh ok sweet!!! Thanks looking into that!!!